pkgs/kde/gear/kdenlive/default.nix

```diff
@@ -12,6 +13,8 @@
   libv4l,
   frei0r,
   fetchpatch,
+  glaxnimate,
+  enableGlaxnimate ? true,
```

Comment:
Again, does this really need to be conditional?

Reply:
Sorry, I missed your first message, you are right I can make itnon optional. I also realized (thanks oborg) that my new commit fails to compile flowblade, so wait before merging.

Reply:
> that my new commit fails to compile flowblade

Is this issue resolved?

Reply:
Fixed